frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -234,20 +237,72 @@ export const AboutResourcePage = ({
           }
           onChangeValue={handleChangeTranslationValues}
           usesTextArea={translationType === 'description'}
+          showErrors={showAllErrors}
         />
       </div>
     );
   };
 
+  /**
+   * Maps the language key to the text
+   */
+  const getLanguage = (val: 'nb' | 'nn' | 'en') => {
+    if (val === 'nb') return 'Bokmål';
+    if (val === 'nn') return 'Nynorsk';
+    return 'Engelsk';
+  };
+
+  /**
+   * Gets the correct text to display for input fields with missing value
+   *
+   * @param val the value
+   * @param isTitle if the field is title or description
+   */
+  const getMissingInputLanguage = (val: LanguageStringType, isTitle: boolean) => {
+    const valArr: ('nb' | 'nn' | 'en')[] = [];
+    const type = isTitle ? 'tittel' : 'beskrivelse';
+
+    // Add the different languages
+    if (val.nb === '') {
+      valArr.push('nb');
+    }
+    if (val.nn === '') {
+      valArr.push('nn');
+    }
+    if (val.en === '') {
+      valArr.push('en');
+    }
+
+    // Return different messages based on the length
+    if (valArr.length === 1) {
+      return `Du mangler oversettelse for ${type} på ${getLanguage(valArr[0])}.`;
+    }
+    if (valArr.length === 2) {
+      return `Du mangler oversettelse for ${type} på ${getLanguage(valArr[0])} og
+      ${getLanguage(valArr[1])}.`;
+    }
+    if (valArr.length === 3) {
+      return `Du mangler oversettelse for ${type} på ${getLanguage(valArr[0])},
+      ${getLanguage(valArr[1])} og ${getLanguage(valArr[2])}.`;
+    }
+    return '';
+  };
+
   /**
    * Displays the content on the page
    */
   const displayContent = () => {
     return (
       <>
-        <h1 className={classes.pageHeader}>Om ressursen</h1>
-        <h2 className={classes.subHeader}>Ressurs type</h2>
-        <p className={classes.text}>Velg ett alternativ fra listen under</p>
+        <Heading size='large' spacing level={1}>
+          Om ressursen
+        </Heading>
+        <Label size='medium' spacing>
+          Ressurs type
+        </Label>
+        <Paragraph short size='small'>
+          Velg ett alternativ fra listen under
```

Comment:
```suggestion
          Velg et alternativ fra listen under
```
Unless you meant to specify the number as one (1)